.github/workflows/compare-apk.yml

```diff
@@ -0,0 +1,72 @@
+name: Compare APK
+on:
+  push:
```

Comment:
Does this need to run on push?

Reply:
If that's not necessary, we could perform the action on pull request.